src/Sarif.UnitTests/TestData/InsertOptionalDataVisitor/ExpectedOutputs/CoreTests_TextFiles.sarif

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2018-11-28",
-  "version": "2.0.0-csd.2.beta.2018-11-28",
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
```

Comment:
>http://json.schemastore.org/sarif-2.0.0 [](start = 14, length = 39)

need a version that forces the prerelease transform

Reply:
Got it.

---
In reply to: [249130020](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249130020) [](ancestors = 249130020)